scripts/validate/govcms-validate-modules

```diff
@@ -11,10 +11,19 @@ set -euo pipefail
 
 GOVCMS_CONFIG_FOLDER_PATH=${GOVCMS_CONFIG_FOLDER_PATH:-"config"}
 GOVCMS_CORE_EXTENTION_FILE_NAME=${GOVCMS_CORE_EXTENTION_FILE_NAME:-"core.extension.yml"}
-GOVCMS_DISALLOWED_MODULES=${GOVCMS_DISALLOWED_MODULES:-"update,dblog"}
```

Comment:
Removed this as it causes a potential vector where someone could override the disallowed module list, we should just hard code this one.